src/public/index.html

```diff
@@ -15,13 +15,15 @@
     </form>
     <div id="success">Successfully logged in!</div>
     <div id="error">Failed to log in</div>
+    <form id="continue-form" method="post" onsubmit="return false;">
```

Comment:
The `onsubmit` attribute here would likely trigger security scans. Anything that we build should avoid HTML attributes like this that contain JS. Using the event object within the event handler with `event.preventDefault()` is the better, more secure alternative.